app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,23 @@
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+ 
+  def perform(initiator, benchmark_type, benchmark_result_type, category)
+    commit_objects = BenchmarkRun.where(benchmark_type: benchmark_type,
```

Comment:
You are fetching benchmark runs here but the variable is named `commit_objects` which is pretty confusing